src/sentry/search/events/constants.py

```diff
@@ -31,6 +31,29 @@
 MEASUREMENTS_FRAMES_FROZEN_RATE = "measurements.frames_frozen_rate"
 MEASUREMENTS_STALL_PERCENTAGE = "measurements.stall_percentage"
 
+VITAL_THRESHOLDS = {
```

Comment:
Nit: Typing this with a typed dict might be nice to ensure you have `poor` and `meh` defined